Missing form types (#1719)
PrestaShop/docs: development/components/form/types-reference/typeahead-product-collection-type.md

---
title: TypeaheadProductCollectionType
---

# TypeaheadProductCollectionType

> This form class is responsible to create a product, with or without attribute field.

Reviewer: ```suggestion
This form class allows you to select a product, with or without an attribute field.
```

Reviewer: again, creating or selecting?
